ci/deps/actions-310.yaml

```diff
@@ -24,39 +24,41 @@ dependencies:
   - pytz
 
   # optional dependencies
-  - beautifulsoup4>=4.11.1
-  - blosc>=1.21.0
-  - bottleneck>=1.3.4
-  - fastparquet>=0.8.1
-  - fsspec>=2022.05.0
+  - beautifulsoup4>=4.11.2
+  - blosc>=1.21.3
+  - bottleneck>=1.3.6
+  - fastparquet>=2022.12.0
+  - fsspec>=2022.11.0
   - html5lib>=1.1
   - hypothesis>=6.46.1
-  - gcsfs>=2022.05.0
+  - gcsfs>=2022.11.0
   - jinja2>=3.1.2
-  - lxml>=4.8.0
-  - matplotlib>=3.6.1, <3.8
-  - numba>=0.55.2
-  - numexpr>=2.8.0
+  - lxml>=4.9.2
+  - matplotlib>=3.6.3, <3.8
+  - numba>=0.56.4
+  - numexpr>=2.8.4
   - odfpy>=1.4.1
-  - qtpy>=2.2.0
-  - openpyxl>=3.0.10
-  - pandas-gbq>=0.17.5
-  - psycopg2>=2.9.3
-  - pyarrow>=7.0.0
+  - qtpy>=2.3.0
+  - openpyxl>=3.1.0
+  # Doesn't install well with pyarrow
+  # https://github.com/pandas-dev/pandas/issues/55525
+  # - pandas-gbq>=0.19.0
```

Comment:
Does this not affect test coverage?

Reply:
Not really. There's only one unit test that just attempts to import this package which doesn't seem useful